agenda-webapps/src/main/resources/locale/portlet/Agenda_en.properties

```diff
@@ -114,3 +114,19 @@ agenda.details.header.menu.delete=Delete
 agenda.details.header.menu.export=Export
 
 agenda.title.addEventRecurrent=Custom recurrence
+
+agenda.settings.drawer.button.save=Save
```

Comment:
For generic labels like "Save", "cancel" .. please use a generic label in stead of duplicating it and creating a new one for each UI. You have already a "Cancel" translation here https://github.com/exoplatform/agenda/pull/26/files#diff-5627e09f74240cab54a6f4d0888af1e1R6 for example

Reply:
okay